client/src/HowItWorks.tsx

```diff
@@ -1,7 +1,341 @@
+                    </Grid>
+                </Grid>
+            </Box>
+
+            <Box className={`${classes.nonprofitContainer} ${classes.mainPageSection} ${classes.limitWidth}`}>
+                <InstructionGrid instructionList={ tabSelected === "nonprofit" ? nonprofitInstructionList : citizenInstructionList }></InstructionGrid>
+            </Box>
+
+            <Box className={`${classes.greySection} ${classes.mainPageSection}`}>
+                <Box className={`${classes.limitWidth}`}>
+                    <Typography variant="h3" component="h3" align="left">
+                        { tabSelected === "nonprofit" ? nonprofitBulletList.title : citizenBulletList.title }
+                    </Typography>
+                    <BulletGrid list={tabSelected === "nonprofit"? nonprofitBulletList.list : citizenBulletList.list}></BulletGrid>
+                </Box>
+            </Box>
+
+            <Box className={`${classes.mainPageSection}`}>
+                <Box className={`${classes.limitWidth} ${classes.questionSection}`}>
+                    <Typography variant="h3" component="h3" align="left">
+                        Still have questions?
+                    </Typography>
+                    <Box className={classes.questions}>
+                        <QuestionList questionList={tabSelected === "nonprofit" ? nonprofitQuestions : citizenQuestions}></QuestionList>
+                    </Box>
+                </Box>
+            </Box>
+
+        </>
+    );
 }
 
+
+
+
+// SUB-COMPONENT InstructionGrid
+
+const instructionStyles = makeStyles<Theme, InstructionProps> ({
+    headerText: {
+        width: '100%',
+        maxWidth: '1100px',
+        fontSize: '1.3rem',
+        textAlign: 'left',
+    },
+    gridBox: {
+        width: '100%',
+        overflow: 'hidden',
+        padding: '20px',
+        '& img': {
+            maxWidth: '100%',
+            height: 'auto',
+        },
+        '& h3': {
+            paddingBottom: '1em',
+        },
+    },
+});
+
+type InstructionProps = {
+    instructionList: { title: string, body: string, image: string }[],
+};
+
+function InstructionGrid(props: InstructionProps) {
+    const classes = instructionStyles(props);
+    var row = 0;
+
+    return (
+        <Grid container justify='space-between'>
+            {props.instructionList.map((instructionItem) => {
+                row++;
+                var text = (
+                    <Grid item sm={6} xs={12} className={`${classes.gridBox}`}>
+                        <Typography variant="h3" component="h3" align="left">
+                            {instructionItem.title}
+                        </Typography>
+                        <Typography variant="body1" component="div" align="left" className={`${classes.headerText}`}>
+                            {instructionItem.body}
+                        </Typography>
+                    </Grid>
+                );
+                var image = (
```

Comment:
i would say pull `text` and `image` out to be separate components as well. these can live in the same file as `InstructionGrid` since they're pretty small

Reply:
Sounds good